addons/l10n_fi/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,548 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <!-- Tags for accounts -->
+    <record id="account_tag_development_costs" model="account.account.tag">
```

Comment:
It seems like we have more tags than needed for the reports...

Reply:
What do you mean ? Which ones do we keep ? I have no idea.

Reply:
Indeed.  Just maybe people could get lost, but ok.